### PR TITLE
eventForm and profilePage with styles

### PR DESCRIPTION
**Pull request by hnekhniadovich**
sketch2
sketch3

### CODE REVIEW

**Comment by Cng000** [comment]
Loving the profile page!

File: frontend/src/components/eventForm/eventForm.js [hunk 1]
```diff
@@ -77,113 +75,129 @@ const temporalInputField = ({
...
+            <Typography style={{fontFamily: "Raleway", fontWeight: '600'}}
+              variant='display1'>studY</Typography>
+          </Grid>
+          <Grid item xs='6'></Grid>
+          <Grid item xs='4' style={{paddingRight: "20px"}}> 
+            <FormControl fullWidth margin='normal'>
+              <Button
+                type='submit'
+                variant="contained"
+                children='Save Event'
```
Comment (Rswaby) [inline frontend/src/components/eventForm/eventForm.js]: I think 'create Event' would be describe the action better rather than 'save Event' but it's fine

Comment (hnekhniadovich) [reply frontend/src/components/eventForm/eventForm.js]: Thanks for the feedback! May be you are right about 'create Event'. I will change it upon my next PL.

**Comment by drsh0x** [comment]
nice!

**Pull request merged**